app/views/contexts/list_collector_context.py

```diff
@@ -1,32 +1,30 @@
+from functools import partial
+
 from flask import url_for
 from flask_babel import lazy_gettext
-from functools import partial
 
 from app.views.contexts.context import Context
-from app.views.contexts.question import build_question_context
 
 
 class ListCollectorContext(Context):
-    def get_item_title(self, list_block_summary, list_item_id, is_primary):
-        rendered_summary = self._placeholder_renderer.render(
-            list_block_summary, list_item_id
-        )
-
-        if is_primary:
-            rendered_summary["item_title"] += lazy_gettext(" (You)")
-
-        return rendered_summary["item_title"]
+    def __call__(self, list_collector_block, return_to=None):
+        return {
+            "list": {
+                "list_items": list(
+                    self._build_list_items_summary_context(
+                        list_collector_block, return_to=return_to
+                    )
+                ),
+                "editable": True,
```

Comment:
`editable` should be passed in and used to determine if the edit/remove links should be added to the context.

Reply:
Should the default be `True`?